src/util/logging.cpp

```diff
@@ -28,6 +28,8 @@ QMutex s_mutexStdErr;
 
 // The file handle for Mixxx's log file.
 QFile s_logfile;
+qint64 s_logMaxFileSize = mixxx::kLogMaxFileSizeDefault;
+bool s_logMaxFileSizeReached = false;
```

Comment:
In spirit of continuing the `tsan` discussion, either this needs to be atomic or the early check below must be moved below the mutex acquire, right?